main.c

```diff
@@ -229,6 +233,33 @@ static bool check_timeouts(struct timeval *t)
 	return tmin > 0;
 }
 
+size_t win_title_get(char *buf, int len)
+{
+	char *argv[8];
+	spawn_t pfd;
+	char w[12] = "", h[12] = "", z[12] = "", fidx[12], fcnt[12];
+	ssize_t n = -1;
+
+	if (wintitle.f.err || buf == NULL || len <= 0)
+		return 0;
+
+	if (mode == MODE_IMAGE) {
+		snprintf(w, ARRLEN(w), "%d", img.w);
+		snprintf(h, ARRLEN(h), "%d", img.h);
+		snprintf(z, ARRLEN(z), "%d", (int)(img.zoom * 100));
+	}
+	snprintf(fidx, ARRLEN(fidx), "%d", fileidx+1);
```

Comment:
These made me realise we don't have constant formatting for operators. I'll create a `clang-format` config based on the current code we have and open a PR, we can discuss further there.

Reply:
@XPhyro Not entirely sure what you mean by consistent formatting for operators tbh. Can you elaborate?

Reply:
Sure. The codebase uses all of `x*5`, `x * 5`, `x+5`, `x + 5`, `(X)*5` and `(X) * 5`.